lighthouse-core/test/audits/first-meaningful-paint-test.js

```diff
@@ -42,88 +45,51 @@ describe('Performance: first-meaningful-paint audit', () => {
 
     it('finds the expected fMP', () => {
       assert.equal(fmpResult.displayValue, '1,100\xa0ms');
-      assert.equal(fmpResult.rawValue, 1099.5);
-    });
-
-    it('finds the correct fMP timings', () => {
-      assert.equal(fmpResult.extendedInfo.value.timings.fMP, 1099.523);
-    });
-
-    it('exposes the FCP timing', () => {
-      assert.equal(fmpResult.extendedInfo.value.timings.fCP, 461.901);
-    });
-
-    it('exposes the navStart timestamp', () => {
-      assert.equal(fmpResult.extendedInfo.value.timestamps.navStart, 668545382727);
+      assert.equal(fmpResult.rawValue, 1099.523);
     });
 
     it('scores the fMP correctly', () => {
       assert.equal(fmpResult.score, 0.99);
     });
-
-    it('did not fall back to an FMP candidate event', () => {
-      assert.ok(!fmpResult.extendedInfo.value.fmpFellBack);
-    });
   });
 
   describe('finds correct FMP', () => {
     it('if there was a tracingStartedInPage after the frame\'s navStart', () => {
       const artifacts = generateArtifactsWithTrace(lateTracingStartedTrace);
-      return FMPAudit.audit(artifacts, {options}).then(result => {
+      return FMPAudit.audit(artifacts, context).then(result => {
         assert.equal(result.displayValue, '530\xa0ms');
-        assert.equal(result.rawValue, 529.9);
```

Comment:
by "these tests" do you mean the assertions on the correct timestamp/navstart/fcp rather than just the FMP? AFAICT I didn't regress coverage of any test scenarios

Reply:
yes at the level where the problematic logic actually was, trace-of-tab :)
https://github.com/AkshayIyer12/lighthouse/blob/f313424de4f2284392aa758d28416c51f7f59dd2/lighthouse-core/test/gather/computed/trace-of-tab-test.js#L13-L18

Reply:
ha, well that's perfect then